src/Wpf.Ui/Controls/Navigation/NavigationBackButton.cs

```diff
@@ -0,0 +1,33 @@
+﻿#nullable enable
+using System.ComponentModel;
+using System.Windows;
+using Wpf.Ui.Controls.Interfaces;
+
+namespace Wpf.Ui.Controls.Navigation;
+
+/// <summary>
+/// Inherited from the <see cref="Wpf.Ui.Controls.Button"/>, used to navigate backwards/>.
+/// </summary>
+public class NavigationBackButton : Button
```

Comment:
Here I would use `System.Windows.Controls.Button` because we don't need the over-complicated `Wpf.Ui.Controls.Button` control.